src/main/java/org/tron/common/runtime/vm/program/ProgramResult.java

```diff
@@ -53,6 +54,14 @@ public void refundGas(long drops) {
     dropUsed -= drops;
   }
 
+  public void setContractAddress(byte[] contractAddress) {
+    this.contractAddress = contractAddress;
+  }
+
+  public byte[] getContractAddress() {
+    return this.contractAddress;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Return a copy of "contractAddress". ([squid:S2384](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2384))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1153&issues=AWSimPJkl6vpliCei5Fv&open=AWSimPJkl6vpliCei5Fv)

[ISSUE_KEY]: AWSimPJkl6vpliCei5Fv